gcloud-java-compute/src/main/java/com/google/gcloud/compute/ForwardingRuleId.java

```diff
@@ -0,0 +1,65 @@
+/*
+ * Copyright 2016 Google Inc. All Rights Reserved.
+ *
+ * Licensed under the Apache License, Version 2.0 (the "License");
+ * you may not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ *
+ *       http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ */
+
+package com.google.gcloud.compute;
+
+/**
+ * Interface for Google Compute Engine forwarding rule identities.
+ */
+public interface ForwardingRuleId {
+
+  /**
+   * Possible types for a Google Compute Engine forwarding rule identity.
+   */
+  enum Type {
+    /**
+     * Global forwarding rules are used to forward traffic to the correct load balancer for HTTP(S)
```

Comment:
It refers to the same concept but for instance operations can be GLOBAL, REGION or ZONE while addresses are only GLOBAL or REGION. There also are other resources that can either be GLOBAL or ZONE. I prefer having a dedicated enum for each identity class rather than having three common enums for possible combinations of types.